#!/usr/bin/env python
PACKAGE = "smoother"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lin_max_vel", double_t, 0, "1", 0.7, 0, 1)
gen.add("lin_min_vel", double_t, 0, "2", 0.7, 0, 1)
gen.add("lin_acc", double_t, 0, "3", 0.25,0,1)
gen.add("lin_decc", double_t, 0, "4", 0.4,0,1)

gen.add("ang_max_speed", double_t, 0, "5", 2.0,0,2.5)
gen.add("ang_acc", double_t, 0, "6", 0.9,0,2)



exit(gen.generate(PACKAGE, "smoother_node", "Smoother"))